evap/staff/templates/staff_semester_view.html

```diff
@@ -384,11 +384,11 @@ <h3>
                                             {{ course.responsibles_names }}
                                         </div>
                                     </td>
-                                    <td data-col="evaluation-count" data-order="{{ course.evaluations.count }}">
-                                        {% if course.evaluations.count == 0 %}
+                                    <td data-col="evaluation-count" data-order="{{ course.evaluations.all|len }}">
+                                        {% if course.evaluations.all|len == 0 %}
```

Comment:
just stumbled across people on the internet claiming that `prefetch_related` combined with a `.count()` call will use the prefetched object cache instead of hitting the database again. This is not what I experienced in the past, but I will have to check again.